src/pages/list/search/projects/index.tsx

```diff
@@ -0,0 +1,180 @@
+import { Form } from '@ant-design/compatible';
+import { Card, Col, List, Row, Select, Typography } from 'antd';
+import React, { Component } from 'react';
+
+import { Dispatch } from 'redux';
+import { FormComponentProps } from 'antd/es/form';
+import { connect } from 'dva';
+import moment from 'moment';
+import AvatarList from './components/AvatarList';
+import { StateType } from './model';
+import { ListItemDataType } from './data.d';
+import StandardFormRow from './components/StandardFormRow';
+import TagSelect from './components/TagSelect';
+import styles from './style.less';
+
+const { Option } = Select;
+const FormItem = Form.Item;
+const { Paragraph } = Typography;
+
+interface ProjectsProps extends FormComponentProps {
+  dispatch: Dispatch<any>;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=4583897)